File: 2025-08-23/unnatural_prime.py
def is_unnatural_prime(n):
    n = abs(n)

    if n == 0 or n == 1:
        return False
    else:
        for i in range(2, int(n / 2)):
            if n % i == 0:
                return False

    return True

print(is_unnatural_prime(1))
print(is_unnatural_prime(-1))
print(is_unnatural_prime(19))
print(is_unnatural_prime(-23))
print(is_unnatural_prime(0))
print(is_unnatural_prime(97))
print(is_unnatural_prime(-61))
print(is_unnatural_prime(99))
print(is_unnatural_prime(-44))

File: 2025-08-23/unnaturalPrime.js
function isUnnaturalPrime(n) {
  n = Math.abs(n);

  if (n == 0 || n == 1) 
    return false;
  else
    for (let i = 2; i <= n / 2; i ++) {
      if (n % i == 0)
        return false;
    }

  return true;
}

console.log(isUnnaturalPrime(1));
console.log(isUnnaturalPrime(-1));
console.log(isUnnaturalPrime(19));
console.log(isUnnaturalPrime(-23));
console.log(isUnnaturalPrime(0));
console.log(isUnnaturalPrime(97));
console.log(isUnnaturalPrime(-61));
console.log(isUnnaturalPrime(99));
console.log(isUnnaturalPrime(-44));